src/enzo/Make.mach.linux-gnu

```diff
@@ -36,8 +36,8 @@ MACH_FILE  = Make.mach.linux-gnu
 # Install paths (local variables)
 #-----------------------------------------------------------------------
 
-LOCAL_HDF5_INSTALL    = /PATH/TO/SERIAL-HDF5/INSTALL # mandatory
-LOCAL_GRACKLE_INSTALL = /PATH/TO/GRACKLE/INSTALL # optional
+LOCAL_HDF5_INSTALL    = /home/dskinner6/software/yt-conda/pkgs/hdf5-1.10.2-hc401514_3# mandatory
```

Comment:
Revert this changes to exclude your directories. You can put a copy of your machine file in `~/.enzo/` that overrides any machine files in the enzo/ directory.